0000-catalog.md

```diff
@@ -359,104 +414,74 @@ CatalogProduct(
     ])
 ```
 
-## Transaction Payload and Execution 
+## Transaction Payload and Execution
 
-The following payloads contain an action enum and the associated action payload. 
+The following payloads contain an action enum and the associated action
+payload.
 This allows for the action payload to be dispatched to the appropriate logic.
-Only the defined actions (enum) are available, and only one Action should be 
+Only the defined actions (enum) are available, and only one Action should be
 performed in a payload.
 
 ### CatalogPayload Transaction
 
 ```
-message CatalogPayload { 
-    enum Actions { 
-        UNSET_ACTION = 0; 
-        CATALOG_CREATE = 1; 
-        CATALOG_UPDATE = 2; 
-        CATALOG_DELETE = 3; 
-    }
-
-    Action action = 1;
-
-    // Approximately when transaction was submitted, as a Unix UTC timestamp
-    uint64 timestamp = 2;
-
-    CatalogCreateAction catalog_create = 3; 
-    CatalogUpdateAction catalog_update = 4; 
-    CatalogDeleteAction catalog_delete = 5; 
-} 
-```
-
-### CatalogProductPayload Transaction
-CatalogProductPayload contains an action enum and the associated action payload. 
-This allows for the action payload to be dispatched to the appropriate logic.
-Only the defined actions are available and only one action payload should be 
-defined in the CatalogProductPayload.
-
-```
-message CatalogProductPayload { 
-    enum Actions { 
-        UNSET_ACTION = 0; 
-        CATALOG_PRODUCT_CREATE = 1; 
-        CATALOG_PRODUCT_UPDATE = 2; 
-        CATALOG_PRODUCT_DELETE = 3; 
-    }
-
-    Action action = 1;
-
-    // Approximately when transaction was submitted, as a Unix UTC timestamp
-    uint64 timestamp = 2;
-
-    CatalogProductCreateAction catalog_product_create = 3; 
-    CatalogProductUpdateAction catalog_product_update = 4; 
-    CatalogProductDeleteAction catalog_product_delete = 5; 
-} 
-```
-
-```
-message CatalogProductOperationPayload { 
-    enum Actions { 
+message CatalogPayload {
+    enum Actions {
         UNSET_ACTION = 0;
-        SET_CATALOG_PRODUCT_STATUS = 1;
+        CATALOG_CREATE = 1;
+        CATALOG_UPDATE = 2;
+        CATALOG_DELETE = 3;
+        CATALOG_PRODUCT_CREATE = 4;
+        CATALOG_PRODUCT_UPDATE = 5;
+        CATALOG_PRODUCT_DELETE = 6;
+         SET_CATALOG_PRODUCT_STATUS = 7;
```

Comment:
This should be CATALOG_PRODUCT_SET_STATUS

Reply:
✅